extensions/amp-story/amp-story.md

```diff
@@ -156,6 +193,22 @@ The `amp-story` component represents an entire story.  The component itself  imp
 
 Identifies that the AMP document is a story.
 
+##### title [required]
+
+The title of the story.
+
+##### publisher [required]
+
+The name of the story's publisher.
+
+##### publisher-logo-src [required]
+
+A publisher's logo in square format (1x1 aspect ratio).
+
+##### poster-portrait-src [required]
+
+The story poster in portrait format (3x4 aspect ratio).
```

Comment:
Can you expand on this for new users.  What would this typically be?  For example,

"The poster is an image that displays in the UI until your story is loaded. The poster can generally be the first screen of your story, although any image that is representative of the story."

Reply:
Added a new section and linked to it.